TaskfMRIAnalysis/scripts/makeSubjectTaskSummary.sh

```diff
@@ -201,19 +206,24 @@ if [ "$VolumeBasedProcessing" = "YES" ] ; then
 	Analyses+="StandardVolumeStats "; # space character at end to separate multiple analyses	
 fi
 
-if [[ "${SummaryName}" = "NONE" || "${SummaryName}" = "" ]]; then
+if [[ "${SummaryDir}" = "NONE" && "${SummaryPrefix}" != "NONE" ]]; then
+    log_Err_Abort "--summarydir=NONE, so --summaryprefix should be NONE as well."
```

Comment:
This case would get overwritten by the below logic anyway, and the converse isn't tested for.